eng/common/pipelines/templates/jobs/prepare-pipelines.yml

```
@@ -23,7 +23,7 @@ jobs:
     name: azsdk-pool-mms-ubuntu-2004-general
     vmImage: MMSUbuntu20.04
   steps:
-    - template: install-pipeline-generation.yml
+    - template: /eng/common/pipelines/templates/steps/install-pipeline-generation.yml
```

The issue is it was moved to jobs.

```
```
```suggestion
    - template: /eng/common/pipelines/templates/jobs/install-pipeline-generation.yml
```

---

@konrad-jamrozik looks like your PR broke a relative path. I think this should still be under steps but we need the full path like @ckairen has in his PR. We didn't, nor should we, move the install step to jobs.